src/vs/workbench/services/files/node/fileService.ts

```diff
@@ -913,7 +913,7 @@ export class FileService implements IFileService {
 		const absolutePath = this.toAbsolutePath(resource);
 
 		return pfs.stat(absolutePath).then(stat => {
-			return new StatResolver(resource, stat.isDirectory(), stat.mtime.getTime(), stat.size, this.options.verboseLogging ? this.options.errorLogger : void 0);
+			return new StatResolver(resource, false, stat.isDirectory(), stat.mtime.getTime(), stat.size, this.options.verboseLogging ? this.options.errorLogger : void 0);
```

Comment:
This is a bit weird, why not introduce a new method `pfs.statLink` that is implemented like this:
* executes `lstat`
* executes `stat` only if the stat is a symbolic link
* returns something like `{ stat: stat, isSymbolicLink: boolean }`

This method could then also be used from within the resolver